src/pds/registrysweepers/legacy_registry_sync/legacy_registry_sync.py

```diff
@@ -65,5 +65,5 @@ def run(
         if not ok:
             log.error(item)
 
-        if dev_mode:
-            break
+        # if dev_mode:
```

Comment:
When code is commented-out, it's helpful to have a comment explaining why

Either that, or just delete the code

Reply:
Thanks, good catch @nutjob4life . Those lines were commented because of my local tests, they need to be uncommented actually, that is done.